bin/annotate

```diff
@@ -178,6 +178,10 @@ OptionParser.new do |opts|
     ENV['ignore_columns'] = regex
   end
 
+  opts.on('-I', '--ignore-routes REGEX', "don't annotate routes that match a given REGEX (i.e., `annotate -I '(mobile|resque|pghero)'`" ) do |regex|
```

Comment:
It looks like "-I" is already used.  Can we just use the long form to avoid conflict?